lib/spack/spack/store.py

```diff
@@ -70,7 +74,34 @@ def parse_install_tree(config_dict):
     #     projections:
     #       all: '{name}-{version}'
 
-    install_tree = config_dict.get('install_tree', {})
+    # Dictionaries of all install trees
+    install_trees = spack.config.get('config:install_trees')
+    shared_install_trees = spack.config.get('config:shared_install_trees')
+
+    # Tests if non-default install root is specified
+    if install_root:
+        # Determines if install_root exists
+        if install_root in install_trees:
+            install_tree = install_trees[install_root]
+        elif shared_install_trees and (install_root in shared_install_trees):
+            install_tree = shared_install_trees[install_root]
+        else:
+            # TODO: provide the user an option to create a new install tree
```

Comment:
Yes that comment is outdated as far as I know. We can initialize pointers to a defined upstream, but it has to be defined first in the config file. A future PR could add a way to do it via a command, but as it stands right now I do like how it is currently. I'll remove this comment though since it is a bit outdated.